Sources/Geotagger/ImageIO/ImageIOGeotaggingItem.swift

```
@@ -41,29 +43,77 @@ public struct ImageIOGeotaggingItem: GeotaggingItemProtocol {
         }
     }
     
-    public func skip(with error: Error) {}
+    public func skip(with error: Error) {
+        guard timeAdjustmentSaveMode == .all,
+              (timeOffset != nil || timezoneOverride != nil) else {
+            return
+        }
+        
+        Task {
```

We don't need Task here same as in `apply`. Also, let's update skip method to throw errors, we don't want to silence errors at this level.